app/javascript/controllers/pair_requests_form_controller.js

```diff
@@ -0,0 +1,62 @@
+import { Controller } from "@hotwired/stimulus"
+export default class extends Controller {
+  static targets = ["click", "invitee", "invitee_tz"]
+  connect(){
+    localStorage.clear()
```

Comment:
Upon further reflection, this is definitely a heavy handed approach 😅. I need to do a better review of stimulus values so I can utilize that instead of localStorage.